### PR TITLE
Minor fixes and possible opensearch fix

### PR DESCRIPTION
**Pull request by godber**
This branch tries to fix some minor deprecation noise and other things like that.  I don't think it really has much of a chance to fix the opensearch timeout bug.

### CODE REVIEW

File: packages/scripts/src/helpers/config.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const OPENSEARCH_HOSTNAME = process.env.OPENSEARCH_HOSTNAME || HOST_IP;
 export const OPENSEARCH_PORT = process.env.OPENSEARCH_PORT || '49210';
 export const OPENSEARCH_USER = process.env.OPENSEARCH_USER || 'admin';
 export const OPENSEARCH_PASSWORD = process.env.OPENSEARCH_PASSWORD || 'admin';
-export const OPENSEARCH_VERSION = process.env.OPENSEARCH_VERSION || '1.3.0';
+export const OPENSEARCH_VERSION = process.env.OPENSEARCH_VERSION || '1.3.6';
```
Comment (godber) [inline packages/scripts/src/helpers/config.ts:65]: The hope is that this version doesn't fail, the rationale for accepting this change even if it doesn't make a difference is that this version is used more.

**Comment by godber** [comment]
refs #3363

**Comment by godber** [comment]
Oh, I should probably fork this off Jared's branch.

File: packages/scripts/src/helpers/test-runner/services.ts [hunk 2]
```diff
@@ -355,8 +355,7 @@ async function checkRestrainedOpensearch(
             const actual: string = body.version.number;
             const expected = options.opensearchVersion;
 
-            const satifies = semver.satisfies(actual, `^${expected}`);
```
Comment (godber) [inline packages/scripts/src/helpers/test-runner/services.ts:358]: This variable was misspelled and also unnecessary.

**Review by jsnoble — APPROVED** [review]
LGTM

**Pull request merged**